static/js/vue-cdr-access/tests/unit/viewType.spec.js

```diff
@@ -1,7 +1,7 @@
 import { shallowMount, flushPromises } from '@vue/test-utils'
 import { createRouter, createWebHistory } from 'vue-router';
-import viewType from '@/components/viewType.vue'
-import displayWrapper from "@/components/displayWrapper";
+import viewType from '../../src/components/viewType.vue'
```

Comment:
Why do these two need to be changed to `../..` when it doesn't seem to be needed anywhere else?